unitTests/ui/ResultsPerPageTest.ts

```diff
@@ -7,6 +7,7 @@ import { FakeResults } from '../Fake';
 import { $$ } from '../../src/utils/Dom';
 import { QueryStateModel } from '../../src/Core';
 import { QUERY_STATE_ATTRIBUTES } from '../../src/models/QueryStateModel';
+import { lab } from 'd3';
```

Comment:
Remove ;)

Reply:
Woops, I don't remember adding this line. I'll remove it.